data-products/src/main/scala/org/sunbird/analytics/model/report/AssessmentCorrectionModel.scala

```diff
@@ -44,9 +44,20 @@ object AssessmentCorrectionModel extends IBatchModelTemplate[String,V3Event,Asse
 
   override def postProcess(events: RDD[AssessOutputEvent], config: Map[String, AnyRef])(implicit sc: SparkContext, fc: FrameworkContext): RDD[AssessOutputEvent] = {
     JobLogger.log(s"Total output events: ${events.count()}", None, Level.INFO)
+    dispatchAssessData(events, config)
     events
   }
 
+  def dispatchAssessData(events: RDD[AssessOutputEvent], config: Map[String, AnyRef])(implicit sc: SparkContext, fc: FrameworkContext): Unit ={
+    val maxRequestSize = config.get("max_request_size").getOrElse(1000000)
+    val fileName = config.getOrElse("outputFile", "/mount/data/analytics/tmp/assessment-correction/")
+
+    val filterDF = events.map(f => JSONUtils.serialize(f)).filter(f => f.getBytes.length > maxRequestSize.asInstanceOf[Int])
+    if (filterDF.count() > 0) {
+      FileDispatcher.dispatch(Map("file" -> fileName.asInstanceOf[AnyRef]), filterDF)
```

Comment:
Use OutputDispatcher instead of FileDispatcher...

Reply:
Done